core-bundle/contao/templates/twig/backend/template_studio/operation/create_or_rename_variant.stream.html.twig

```diff
@@ -11,11 +11,11 @@
             {% block title %}{{ ('operation.'~operation~'.0')|trans }}{% endblock %}
 
             {% block content %}
-                <p class="description">
+                <div class="description">
                     {{ ('dialog.'~operation_type~'_variant')|trans([identifier_token])|raw }}
-                </p>
+                </div>
 
-                <div class="template_name widget">
+                <div class="name_container">
```

Comment:
```suggestion
                <div class="template_name name_container">
```

Shouldn't this still include the `template_name` class, so that the identifier/variant fragment/extension are styled exactly like in the editor tab or info block (blue identifier to denote a user template and so on)?